src/olympia/amo/templatetags/jinja_helpers.py

```diff
@@ -424,7 +380,7 @@ def css(bundle, media=False, debug=None):
     """
     urls = get_css_urls(bundle, debug)
     if not media:
-        media = getattr(settings, 'CSS_MEDIA_DEFAULT', 'screen,projection,tv')
+        media = 'all'
 
     return _build_html(urls, '<link rel="stylesheet" media="%s" href="%%s" />' % media)
```

Comment:
if I was writing this today I'd write `return _build_html(urls, f'<link rel="stylesheet" media="{media or "all"}" href="%s" />')` but 🤷